server/assets-v1/templates/src/pages/client_portal/register.html

```diff
@@ -187,18 +185,46 @@ <h1 class="font-bold text-3xl md:text-4xl text-[#3751FE] mb-11 animate-slideFrom
               password: password.value,
             }),
           });
-          alert("Registration successful!");
+          showToastMessage("Registration successful!", "error")
           window.location.href = "[[ .ProxyURL ]]/client-login-page";
         } catch (error) {
           console.error(error);
-          alert("Registration failed!");
+          showToastMessage("Registration failed!", "error")
+        }
+      };
+
+      const showToastMessage = (message, type) => {
+        toastMessage.value = message;
+        showToast.value = true;
+        setTimeout(() => {
+          showToast.value = false;
+        }, 3000);
+      };
+
+      const checkBackendUri = async () => {
+        try {
+          isBUFocused.value = false
+          const response = await fetch("[[ .ProxyURL ]]/api/v1/check-backend-uri", {
+            method: "POST",
+            headers: {
+              "Content-Type": "application/json",
+              "BackendURI": backend_uri.value,
```

Comment:
its not a best practice to put this value to header. If your API method is POST, put it in the request body instead. This article will help you to differentiate the usage of header, request body, and query params https://stackoverflow.com/a/69970738

Reply:
I moved request to body from header in my last changes.